enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/InteractiveDashboard.tsx

```diff
@@ -26,30 +17,45 @@ import {
 import { useSdkDispatch, useSdkSelector } from "embedding-sdk/store";
 import { DASHBOARD_DISPLAY_ACTIONS } from "metabase/dashboard/components/DashboardHeader/DashboardHeaderButtonRow/constants";
 import { useEmbedTheme } from "metabase/dashboard/hooks";
+import type { MetabasePluginsConfig } from "metabase/embedding-sdk/types/plugins";
 import { PublicOrEmbeddedDashboard } from "metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard";
 import type { PublicOrEmbeddedDashboardEventHandlersProps } from "metabase/public/containers/PublicOrEmbeddedDashboard/types";
 import { setErrorPage } from "metabase/redux/app";
 import { getErrorPage } from "metabase/selectors/app";
 import { getEmbeddingMode } from "metabase/visualizations/click-actions/lib/modes";
 import type { ClickActionModeGetter } from "metabase/visualizations/types";
 
+import type { BaseInteractiveQuestionProps } from "../InteractiveQuestion";
+
 import { InteractiveDashboardProvider } from "./context";
 
+/**
+ * @interface
+ * @category InteractiveDashboard
+ */
 export type InteractiveDashboardProps = {
-  className?: string;
-  style?: CSSProperties;
+  /**
+   * Additional mapper function to override or add drill-down menu. See the implementing custom actions section for more details.
+   */
   plugins?: MetabasePluginsConfig;
 
+  // @todo pass the question context to the question view component,
+  //       once we have a public-facing question context.
```

Comment:
Does this get passed to the doc, I assume not. Are only jsdoc comments extracted to the doc?

Reply:
No, it's not passed to generated docs, only js comments